db/data/20221125160837_backfill_cyprus_nationality_for_hesa_trainees.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class BackfillCyprusNationalityForHesaTrainees < ActiveRecord::Migration[6.1]
+  def up
+    trainee_ids = [
+      46509,
+      24830,
+      62709,
+      70447,
+      75420,
+      75191,
+      69538,
+      79556,
+      75340,
+      108613,
+      47011,
+      76904,
+      144223,
+      144050,
+      139322,
+      127131,
+      47393,
+      149521,
+      146568,
+      144047,
+      21798,
+      135081,
+    ]
+
+    eu_cypriot = Nationality.find_by(name: "cypriot (european union)")
+
+    trainee_ids.each do |id|
+      trainee = Trainee.where(id: id).first
+
+      if trainee
+        Nationalisation.find_or_create_by!(trainee_id: id, nationality_id: eu_cypriot)
```

Comment:
have used `eu_cypriot` here instead of `eu_cypriot.id` as thought the ActiveRecord association on nationalisations would be able to pick out the id without me specifying it